openapi/paths/check/redeemOnly.yaml

```diff
@@ -0,0 +1,25 @@
+post:
+  tags:
+  - Redeem
```

Comment:
I'm thinking let's put all 4 of the posts we do related to Checks onto the same tag so they appear on the same C# interface. Maybe make it the "Checks" tag, then it'll be the `IChecks` interface?